pymongoexplain/commands.py

```diff
@@ -81,7 +88,12 @@ def __init__(self, collection: Collection, pipeline, session,
         super().__init__(collection.name)
         self.command_document = {"pipeline": pipeline, "cursor": cursor_options}
         for key, value in kwargs.items():
-            self.command_document[key] = value
+            if key == "batchSize":
+                if value == 0:
```

Comment:
Do we need to check for None too?

Reply:
None values are already removed in the converting to snakecase step